jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -58,7 +58,7 @@ public class ThemeManager {
     private final WorkspacePreferences workspacePreferences;
     private final FileUpdateMonitor fileUpdateMonitor;
     private final Consumer<Runnable> updateRunner;
-    private final ThemeWindowManager themeWindowManager;
+    private ThemeWindowManager themeWindowManager;
```

Comment:
Add `@Nullable`; because we don't use `Optionals` in class variables; do we @Siedlerchr